src/lib.rs

```diff
@@ -390,10 +407,16 @@ fn command<'a>(i: &'a [u8]) -> IResult<&'a [u8], Command<'a>> {
     )
 }
 
+/// An error as returned by a `Handler` method.
 pub enum Error {
-    // The meaning of the value is not defined by the protocol; so it
-    // can be used by a handler for debugging.
+    /// A plain error.  The meaning of the value is not defined by the
+    /// protocol; so it can be used by a handler for debugging.
```

Comment:
Grammatical nit: as much as I love semicolons, I think this would be better as two complete sentences ("...protocol.  Different values can therefore be used by a handler for debugging purposes.") or just using a comma to separate.